src/amo/routes.js

```diff
@@ -5,8 +5,7 @@ import App from './containers/App';
 import Home from './containers/Home';
 
 export default (
-  <Route path="/" component={App}>
+  <Route path="/(:lang/)" component={App}>
```

Comment:
I wonder how this will work if you get something like `/firefox/addons/`, would it be smart enough to see that there is no `lang`? Maybe we shouldn't worry about redirecting the lang other than at `/`?